Blish HUD/GameServices/Input/Mouse/MouseHandler.cs

```diff
@@ -126,9 +128,12 @@ public class MouseHandler : IInputHandler {
         }
 
         public bool HandleInput(MouseEventArgs mouseEventArgs) {
-            if (mouseEventArgs.EventType == MouseEventType.MouseMoved) return false;
+            if (mouseEventArgs.EventType == MouseEventType.MouseMoved) {
+                PositionRaw = new Point(mouseEventArgs.PointX, mouseEventArgs.PointY);
+                return false;
+            }
 
-            if (_cameraDragging && (mouseEventArgs.EventType == MouseEventType.RightMouseButtonReleased))
+            if (_cameraDragging && mouseEventArgs.EventType == MouseEventType.RightMouseButtonReleased)
```

Comment:
I know this doesn't originate in this PR (it originated in Archomeda's when he revamped the InputService), but could you put the braces back on this conditional?  I dislike skipping on the braces for one liners unless it's for an early exit at the top, but this also has an else if that does have braces which makes it exceptionally bleck.

Reply:
I did not remove them by hand, so I assume it is a setting in editorconfig? I can put them back in ofc.

Reply:
Oh, the braces, not the parentheses.